### ISSUE TITLE
GitHub Actions badge in Readme is broken

### ISSUE THREAD

**Issue by atouchet** [smol-rs/polling#142]
![Build](https://github.com/smol-rs/polling/workflows/Build%20and%20test/badge.svg)

I do not see any workflow named Build%20and%20test. Is this a copy and paste error?

**Comment by notgull** [comment]
Yeah, I think this is a copy and paste error. I would accept a PR to fix it.
